core/pkg/hostsensorutils/hostsensor.yaml

```diff
@@ -52,12 +53,19 @@ spec:
         volumeMounts:
         - mountPath: /host_fs
           name: host-filesystem
-        readinessProbe:
+        startupProbe:
           httpGet:
-            path: /kernelVersion
+            path: /readyz
             port: 7888
-          initialDelaySeconds: 1
-          periodSeconds: 1
+          initialDelaySeconds: 10
+          failureThreshold: 10
```

Comment:
obviously you should increase this if you probe every second

Reply:
do you think 30 is a suitable value?

Reply:
yeah, honestly if the machine is so slow that hostscanner takes more than 30s to start we have other issues